### ISSUE TITLE
Editor lag

### ISSUE THREAD

**Issue by opus1269** [PolymerLabs/polymer-atom#9]
I have experienced significant editor lag - scrolling and typing in .html files when polymer-atom is enabled.

I suppose it could be an interaction with another package, but before I try to isolate it, I wanted to check to see if others have experienced this or if it is a known issue.

**Comment by earlyster** [comment]
yes I have faced the same issue and had to disable the plugin and just use command line tool it would be nice if this could only be enabled on save only as well through a setting.

**Comment by garcus** [comment]
I experience the same issue. Agree with @earlyster, would be sufficient on save only.

**Comment by opus1269** [comment]
I wrote [linter-polymer](https://atom.io/packages/linter-polymer) to do this.

**Comment by JGSolutions** [comment]
@opus1269 Seems to be better. I will work with it more. Thanks

Actually tried working with for 5 minutes and get the same lagging

**Comment by abdonrd** [comment]
:+1:

**Comment by JGSolutions** [comment]
What is the progress?

**Comment by opus1269** [comment]
Have you tried re-installing polymer-atom or linter-polymer lately?  The latest release of polylint includes this [PR](https://github.com/PolymerLabs/polylint/pull/99) that should speed things up. New installs will get it. I haven't been doing much Polymer development lately, so I don't know if the improvement is enough to make on the fly linting painless.

I'm going to push out a new linter-polymer soon so existing users gets the update.

**Comment by JGSolutions** [comment]
Yesterday atom.io released an updated so i decided to reinstall everything. I removed the atom settings folder and started from scratch.

I just installed your plugin linter-polymer 0.0.2 & terminal-plus. Began working on the polymer files and 15min later I couldn't work anymore. I checked the memory status for atom and one of the helper files went up to 1 gig of memory usage. Disabled that plugin and atom memory dropped.

When will you push a new linter-polymer update as you mentioned above.

**Comment by jay8t6** [comment]
I am facing the same issue.

**Comment by zinkkrysty** [comment]
I am also having problems with lag in the editor when enabling polymer-atom linter plugin. Basically it renders Atom unusable when working on Polymer projects.
